localization/diagnostics/en.yaml

```diff
@@ -3136,6 +3136,9 @@
 - id: optional_base_not_unwrapped
   msg: "value of optional type %0 must be unwrapped to refer to member %1 of wrapped base type %2"
 
+- id: invalid_optional_infered_keypath_root
+  msg: "key path root inferred as optional type %0 must be chained to refer to member %1 of unwrapped type %2"
```

Comment:
Now that `!.` is offered as a fix, perhaps it should be `optional type %0 must be unwrapped`?

Reply:
Just adjusted :)